modules/rosa-cli-hcp-classic-examples.adoc

```diff
@@ -0,0 +1,254 @@
+                "s3:PutBucketTagging",
+                "s3:PutBucketPolicy",
+                "s3:PutObjectTagging",
+                "s3:PutBucketPublicAccessBlock",
+                "secretsmanager:CreateSecret",
+                "secretsmanager:TagResource"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+----
+
+[id="rosa-list-account-roles-hcp-classic_{context}"]
+== List your account roles
+Run the following command with the specified permissions to list your account roles.
+
+.Input
+[source,terminal]
+----
+$ rosa list account-roles
+----
+.Policy
+[source,json]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "ListAccountRoles",
+            "Effect": "Allow",
+            "Action": [
+                "iam:ListRoleTags",
+                "iam:ListRoles"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+----
+
+[id="rosa-list-operator-roles-hcp-classic_{context}"]
+== List your Operator roles
+Run the following command with the specified permissions to list your Operator roles.
+
+.Input
+[source,terminal]
+----
+$ rosa list operator-roles
+----
+.Policy
+[source,json]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "ListOperatorRoles",
+            "Effect": "Allow",
+            "Action": [
+                "iam:ListRoleTags",
+                "iam:ListAttachedRolePolicies",
+                "iam:ListRoles",
+                "iam:ListPolicyTags"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+----
+
+[id="rosa-list-OIDC-providers-hcp-classic_{context}"]
+== List your OIDC providers
+
+Run the following command with the specified permissions to list your OIDC providers.
+
+.Input
+[source,terminal]
+----
+$ rosa list oidc-providers
+----
+.Policy
+[source,json]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "ListOidcProviders",
+            "Effect": "Allow",
+            "Action": [
+                "iam:ListOpenIDConnectProviders",
+                "iam:ListOpenIDConnectProviderTags"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+----
+
+[id="rosa-verify-quota-hcp-classic_{context}"]
+== Verify your quota
+
+Run the following command with the specified permissions to verify your quota.
+
+.Input
+[source,terminal]
+----
+$ rosa verify quota
+----
+.Policy
+[source,json]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VerifyQuota",
+            "Effect": "Allow",
+            "Action": [
+                "elasticloadbalancing:DescribeAccountLimits",
+                "servicequotas:ListServiceQuotas"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+----
+
+[id="rosa-delete-oidc-config-hcp-classic_{context}"]
+== Delete your OIDC configuration
+
+Run the following command with the specified permissions to delete your OIDC configuration by using `auto` mode.
```

Comment:
Run the following command with the specified permissions to delete Managed OIDC configuration by using `auto` mode.